src/ncp/spinel.h

```diff
@@ -1154,18 +1154,11 @@ typedef enum
     SPINEL_PROP_THREAD_COMMISSIONER_ENABLED
                                         = SPINEL_PROP_THREAD_EXT__BEGIN + 16,
 
-    /// Thread TMF proxy enable
-    /** Format `b`
-     *
-     * Default value is `false`.
-     */
-    SPINEL_PROP_THREAD_TMF_PROXY_ENABLED
-                                        = SPINEL_PROP_THREAD_EXT__BEGIN + 17,
 
-    /// Thread TMF proxy stream
+    /// Thread UDP proxy stream
     /** Format `dSS`
      */
-    SPINEL_PROP_THREAD_TMF_PROXY_STREAM = SPINEL_PROP_THREAD_EXT__BEGIN + 18,
+    SPINEL_PROP_THREAD_UDP_PROXY_STREAM = SPINEL_PROP_THREAD_EXT__BEGIN + 18,
```

Comment:
@bukepo let's define new spinel properties for new UDP stream. 

To ensure compatibility with older version of wpantund, we should keep existing property definitions the same as before. We can add new definitions but should avoid changing format of existing properties or their behavior.

Reply:
Kept the old definition.